.pylintrc

```diff
@@ -71,18 +94,20 @@ disable=C0111,W0611,R0201,W0613,R0903,E1101,I0011,W0232
 #
 # W0232     no-init                 New-style classes (see above) do not require __init__ methods,
 #                                   the object class has a perfectly fine default constructor.
+disable=import-star-module-level,old-octal-literal,oct-method,print-statement,unpacking-in-except,parameter-unpacking,backtick,old-raise-syntax,old-ne-operator,long-suffix,dict-view-method,dict-iter-method,metaclass-assignment,next-method-called,raising-string,indexing-exception,raw_input-builtin,long-builtin,file-builtin,execfile-builtin,coerce-builtin,cmp-builtin,buffer-builtin,basestring-builtin,apply-builtin,filter-builtin-not-iterating,using-cmp-argument,useless-suppression,range-builtin-not-iterating,suppressed-message,no-absolute-import,old-division,cmp-method,reload-builtin,zip-builtin-not-iterating,intern-builtin,unichr-builtin,reduce-builtin,standarderror-builtin,unicode-builtin,xrange-builtin,coerce-method,delslice-method,getslice-method,setslice-method,input-builtin,round-builtin,hex-method,nonzero-method,map-builtin-not-iterating,missing-docstring,unused-import,no-self-use,unused-argument,too-few-public-methods,no-member,locally-disabled,locally-disabled,no-init
```

Comment:
I wonder if we should remove the documentation for the disabled checkers, because now those comments seem to become obsolete (and they seem to contain references to private code btw.).